### PR TITLE
allow mouse pointer to pick against collidable entities

### PR DESCRIPTION
**Pull request by danteruiz**
allow mouse pointer and controller dispatcher pointers to pick against collidable entities

Ticket - https://highfidelity.manuscript.com/f/cases/20510/cannot-click-on-Web-entities

### CODE REVIEW

**Comment by danteruiz** [comment]
Test plan 
- create a web entity in desktop mode
- click on links or button on a web page
- they should also work.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16273/HighFidelity-Beta-PR14661-0dc5e77.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16273/HighFidelity-Beta-PR14661-0dc5e77.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16273/HighFidelity-Beta-Interface-PR14661-0dc5e77.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16273/HighFidelity-Beta-PR14661-0dc5e77.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16273/HighFidelity-Beta-Interface-PR14661-0dc5e77.exe

**Comment by SamGondelman** [comment]
Oooo great find.  The script one actually shouldn’t need to be changed (picks made from scripts should always hit collidable things).  For the other mouse pick, you can get rid of both COLLIDABLE and NONCOLLIDABLE (which will make it pick against both).

**Comment by danteruiz** [comment]
What other mouse pick are you taking about?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16280/HighFidelity-Beta-PR14661-c4c13f7.apk

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -2284,7 +2284,8 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
 
     // Setup the mouse ray pick and related operators
     {
-        auto mouseRayPick = std::make_shared<RayPick>(Vectors::ZERO, Vectors::UP, PickFilter(PickScriptingInterface::PICK_ENTITIES() | PickScriptingInterface::PICK_INCLUDE_NONCOLLIDABLE()), 0.0f, true);
+        auto mouseRayPick = std::make_shared<RayPick>(Vectors::ZERO, Vectors::UP, PickFilter(PickScriptingInterface::PICK_ENTITIES() | PickScriptingInterface::PICK_INCLUDE_NONCOLLIDABLE() |
+                                                                                             PickScriptingInterface::PICK_INCLUDE_COLLIDABLE()), 0.0f, true);
```
Comment (SamGondelman) [inline interface/src/Application.cpp]: You can get rid of both COLLIDABLE and NONCOLLIDABLE here.  If it has neither, it collided against both.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16281/HighFidelity-Beta-PR14661-4b99a79.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16281/HighFidelity-Beta-PR14661-4b99a79.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16281/HighFidelity-Beta-Interface-PR14661-4b99a79.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16281/HighFidelity-Beta-PR14661-4b99a79.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16281/HighFidelity-Beta-Interface-PR14661-4b99a79.exe

**Comment by JeffClinton** [comment]
Currently testing

**Pull request merged**